tests/scripts/run-xunit-perf.py

```
@@ -149,7 +149,7 @@ def run_command(runArgs, environment, errorMessage):
         subprocess.check_output(runArgs, stderr=subprocess.PIPE, env=environment)
     except subprocess.CalledProcessError as e:
         log(e.output.decode('utf-8'))
-        raise RuntimeException(errorMessage);
+        raise RuntimeError(errorMessage);
```

Could you just raise the caught exception?

---

You do not need the `;` at the end of the line

---

Sure.